### ISSUE TITLE
[Bug Report] - Missing annotation for an array property after code generation

### ISSUE THREAD

**Issue by andreadangel1** [nickshoe/asyncapi-php-template#2]
**Environment details**
-PHP version: _8.1.16_
-Message broker: _RabbitMQ_

**Description**
If I have a schema with an array type property, the code generation will not add the annotation for that one into the PHP class. 
JMS will not be able to deserialize the array property during the subscription of the message and will give a PHP Fatal error.

**Example**
I have an object-type schema like this into the asyncapi .yaml file:
```
Email:
      type: object
      properties:
        subject:
          type: string
        body:
          type: string
        recipients:
          type: array
          items:
            type: string
```

From this schema, it will be generated a model called "Email.class.php" with these 3 properties:
```
<?php

namespace AsyncAPI\Models;

class Email
{

  private string $subject;
  private string $body;
  private array $recipients;
...
...
...
```

Now, we publish an event with an "Email" object like this:
```
$event = new EmailEvent(
    new Email(
        'Password expired!',
        'Hi, your password has been expired',
        ['test1@gmail.com','test2@gmail.com']
    ),
    ...
    ...
);
```

Then, If we try to subscribe the message from RabbitMQ, it will give this error:
`PHP Fatal error:  Uncaught JMS\Serializer\Exception\RuntimeException: You must define a type for AsyncAPI\Models\Email::$recipients.`

**Workaround**
Add this code to the class. In this example, it will be the Email class:
```
<?php

namespace AsyncAPI\Models;

use JMS\Serializer\Annotation as Serializer; //this

class Email
{

  private string $subject;
  private string $body;
  /**
  * @Serializer\Type("array<string>")  //and this
  */
  private array $recipients;
...
...
...
```

**Comment by nickshoe** [comment]
Hi @andreadangel1 and thanks!

It seems you've got it all figured out. 😉 

Would you open a PR for this?

**Comment by nickshoe** [comment]
resolved in v0.2.10-0
